buildSrc/src/main/groovy/org/gradle/binarycompatibility/AcceptedApiChanges.groovy

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.binarycompatibility
+
+import groovy.json.JsonOutput
+import groovy.json.JsonSlurper
+import org.gradle.util.GradleVersion
+
+class AcceptedApiChanges {
+
+    GradleVersion baseVersion
+    Map<ApiChange, String> acceptedChanges
```

Comment:
If I have a list I would need to iterate through the list instead of calling `get` to get the accepted reason for an API change.